docs/tutorials/cta_sensitivity.ipynb

```diff
@@ -182,7 +182,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sensitivity_estimator = SensitivityEstimator(gamma_min=5, n_sigma=3)\n",
+    "sensitivity_estimator = SensitivityEstimator(gamma_min=5, n_sigma=3, bkg_syst_fraction=0.10)\n",
```

Comment:
Perhaps it is better to add a line about the `bkg_syst_fraction` in the previous markup cell.